client/src/components/user/HomeScreen/HomeScreen.jsx

```diff
@@ -1,22 +1,37 @@
 import React from "react";
+import { BrowserRouter as Router, Route, Link, Switch, Redirect } from 'react-router-dom';
 
-let userData = {
-  userName: "Jacob",
-  password: "1234",
-  authLevel: 1,
-  name: "Jacob Zielinski"
-};
+import RequestLeave from "../RequestLeave/RequestLeave";
+import RotaTimesheet from "../RotaTimesheet/RotaTimesheet";
+import ClockInOut from "../ClockInOut/ClockInOut"
+import StaffOverview from "../../manager/StaffOverview/StaffOverview";
+import ViewEditUsers from "../../admin/ViewEditUsers/ViewEditUsers"
+
+// let userData = {
+//   userName: "Jacob",
+//   password: "1234",
+//   authLevel: 1,
+//   name: "Jacob Zielinski"
+// };
 
 const HomeScreen = props => {
-  if (props.authLevel === 1) {
+<<<<<<< HEAD
+  if (props.userid === 1) {
+=======
+  if (userData.authLevel === 1) {
+>>>>>>> master
     return (
-      <div>
-        <div>{props.holidayRequest} Holiday Request</div>
-        <div>{props.timeSheet} Timesheet</div>
-        <div>{props.ClockIn} Clock In</div>
-      </div>
+        <div>
+          <div>{props.holidayRequest} <Link to="/holidayreq">Holiday Request</Link></div>
+          <div>{props.timeSheet} <Link to="/timesheet">Timesheet</Link></div>
+          <div>{props.ClockIn} Clock In</div>
+        </div>
     );
-  } else if (props.authLevel === 2) {
+<<<<<<< HEAD
```

Comment:
I don't know why this was changed, because I think the correct code is:
} else if (props.authLevel === 2) {